CHANGELOG.md

```diff
@@ -32,6 +32,183 @@
 - [v0.1.0-rc2](#v010-rc2)
 - [v0.1.0-rc1](#v010-rc1)
 
+# v1.0
+
+On behalf of Kubernetes SIG Network, we are pleased to announce the v1.0 release!
+This release marks a huge milestone for this project. Several key APIs are
+graduating to GA (generally available), while other significant features have
+been added to the Experimental channel.
+
+It's been four years since this project began, and we would never have got here
+without the support of a dedicated and active community. The maintainers would like
+to thanks _everyone_ who's contributed to Gateway API, whether in the form of
+commits to the repo, discussion, ideas, or general support.
+
+We literally couldn't have got this far without you.
```

Comment:
The bottom sentence correlates to the paragraph.

```suggestion
commits to the repo, discussion, ideas, or general support. We literally couldn't have got this far without you.
```

Reply:
One additional nit - I think "got" should be "gotten" here.